atc/worker/worker.go

```diff
@@ -1,3 +1,7 @@
+// This package will eventually evolve to becoming a concrete implementation of a runtime
```

Comment:
```suggestion
// Package worker will eventually evolve to becoming a concrete implementation of a runtime
```

package docs are meant to start with `Package <pkg_name>`, e.g.:

```
// Package io provides basic interfaces to I/O primitives.
// Its primary job is to wrap existing implementations of such primitives,
// such as those in package os, into shared public interfaces that
// abstract the functionality, plus some other related primitives.
```

https://github.com/golang/go/blob/71239b4f491698397149868c88d2c851de2cd49b/src/io/io.go#L5